build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -523,6 +547,14 @@ def filter_and_run_benchmarks(
   return (benchmark_files, captures, errors)
 
 
+def set_frequency_scaling_governor(governor: str):
+  git_root = execute_cmd_and_get_output(["git", "rev-parse", "--show-toplevel"])
+  cpu_script = os.path.join(
+      git_root, "build_tools/benchmarks/set_android_scaling_governor.sh")
```

Comment:
`"build_tools", "benchmarks", "set_android_scaling_governor.sh"`?

Reply:
I think it's fine to have a literal path all in one string. The thing you don't want to do is `some_directory + filename` since then you have to fiddle with whether or not there's a trailing slash in directory names